test/cpp/qps/driver.cc

```diff
@@ -573,6 +574,11 @@ std::unique_ptr<ScenarioResult> RunScenario(
 
   // Start a run
   gpr_log(GPR_INFO, "Starting");
+
+  google::protobuf::Timestamp start_timestamp;
```

Comment:
There is no need to create a timetamp object for start and end. Instead, you can do auto start_time = time(Null) and then update the field in the proto directly at the end.